datamade/just-spaces
# Add about page

## Overview

Closes #199. Creates an About page using text from this UCD document: https://docs.google.com/document/u/1/d/18MsGjBQjqoLFtKZYexaJC_v1N18yAKKVuJJameOzGHI/edit?usp=sharing

About page should be accessible without a login and include pictures taken from UCD's site. This PR also incorporates the body text font from UCD's central branding. 

I grabbed some pictures from UCD's website for this. It would be nice, but more time-intensive, to populate this page with screengrabs from the site itself to show what it looks like. 

![Screen Shot 2019-07-22 at 2 08 06 PM](https://user-images.githubusercontent.com/1094243/61658089-2bb11280-ac8a-11e9-90b6-d8c5f1dc69cd.png)


## Testing Instructions

 * Run the site locally and navigate to `http://localhost:8000/`
* Look over the about page and make sure everything's in order
